EssentialCSharp.Web/Services/SiteMappingService.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace EssentialCSharp.Web.Services;
+
+public class SiteMappingService
```

Comment:
We should probably put this behind an interface for testability. Related we should add some integration test for that controller.

Reply:
Opened https://github.com/IntelliTect-dev/EssentialCSharp.Tooling/issues/881 for the testing part